src/KubeLibrary/KubeLibrary.py

```diff
@@ -569,3 +569,21 @@ def get_ingress_details_in_namespace(self, name, namespace):
         """
         ret = self.extensionsv1beta1.read_namespaced_ingress(name, namespace)
         return ret
+
+    def get_daemonsets_in_namespace(self, namespace, label_selector=""):
+        """Gets a list of available daemonsets.
+        Can be optionally filtered by label. e.g. label_selector=label_key=label_value
+        Returns list of daemonsets.
```

Comment:
Please add docs for all the parameters for all new keywords.

 I know it is not the case for all keywords in KubeLIbrary, but we would need that at some point.

Reply:
Done.